<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloseDialog</class>
 <widget class="QDialog" name="CloseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>85</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Close SR</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../kueue.qrc">
    <normaloff>:/icons/app/kueue32.png</normaloff>:/icons/app/kueue32.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="closeLabel">
     <property name="text">
      <string>Close SR# ?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="closeCombo">
     <item>
      <property name="text">
       <string>Solution Provided</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solution Presumed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Charge Cancelled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Charge Refunded</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Charge Awaiting Engineering</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Awaiting Engineering</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../kueue.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CloseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CloseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
